conda_build/cli/main_build.py

```diff
@@ -503,6 +508,18 @@ def parse_args(args: Sequence[str] | None) -> tuple[ArgumentParser, Namespace]:
 
     parsed = parser.parse_args(args)
     check_recipe(parsed.recipe)
+
+    # TODO: Remove in 25.1
+    if (
+        all(not arg.startswith("--package-format") for arg in args)
+        and warn_about_default_pkg_format
+    ):
+        get_logger(__name__).warning(
+            "The default `pkg_format` value will become '.conda' in 25.1. "
+            "If you want to keep using `.tar.bz2`, consider:\n"
+            "- Setting `conda_build.pkg_format: 'tar.bz2' in your condarc file.\n"
+            "- Using `--pkg-format=tar.bz2` in the CLI.",
+        )
```

Comment:
This also needs an import above from `conda_build.deprecations`:

```suggestion
        deprecated.topic(
            "24.11",
            "25.1",
            topic=f"The default `pkg_format` of '.tar.bz2'",
            addendum=(
                "\n\n"
                "The new default `pkg_format` value will be '.conda'."
                "If you want to keep using `.tar.bz2`, consider:\n"
                "- Setting `conda_build.pkg_format: 'tar.bz2' in your condarc file.\n"
                "- Using `--pkg-format=tar.bz2` in the CLI.\n",
            ),
            deprecation_type=FutureWarning,
        )

```

Reply:
I use this in my condarc
```
conda-build:
  pkg_format: 2
```